src/views/TravelRequest/index.js

```diff
@@ -0,0 +1,118 @@
+import BackImg from "../../assets/images/back2.png";
+import Link from '@material-ui/core/Link';
+
+const TravelRequest = () => {
+  const classes = useStyles();
+  return (
+    <>
+    <Header/>
+    <div className={classes.mainContainer}>
+      <img className={classes.backImage} src={BackImg} alt="back imag" />
+      <Typography
+        variant="body1"
+        style={{
+          fontWeight: "700",
+          fontSize: "2rem",
+          top: "7rem",
+          position: "relative",
+          textAlign:"center",
+        }}
+      >
+        Travel request
+      </Typography>
+      <div className={classes.root}>
+        <Card>
+          <CardHeader className={classes.cardHeader}>
+            <Typography
+              gutterBottom
+              variant="body1"
+              style={{ fontWeight: "700", marginBottom: "0" }}
+            >
+              Reservation Summary
+            </Typography>
+          </CardHeader>
+          <CardContent>
+            <Typography variant="body1" className={classes.fontWeight}>
+              Check in:
+            </Typography>
+            <Typography gutterBottom variant="body2">
+              Monday 12 November 2020
+            </Typography>
+
+            <Typography variant="body1" className={classes.fontWeight}>
+              Check out:
+            </Typography>
+            <Typography gutterBottom variant="body2">
+              Monday 12 November 2020
+            </Typography>
+            <Divider light />
+            <div
+              style={{
+                display: "flex",
+                justifyContent: "space-between",
+                margin: "1rem 0",
+              }}
+            >
+              <Typography variant="body1" className={classes.fontWeight}>
+                Rooms
+              </Typography>
+            </div>
+            <Typography gutterBottom variant="body1">
+              Room #1
+            </Typography>
+            <div style={{ display: "flex", justifyContent: "space-between" }}>
+              <Typography variant="body1" className={classes.fontWeight}>
+                Room charge
+              </Typography>
+              <Typography gutterBottom variant="body1">
+                $300
+              </Typography>
+            </div>
+            <div style={{ display: "flex", justifyContent: "space-between" }}>
+              <Typography variant="body1" className={classes.fontWeight}>
+                Total charge
+              </Typography>
+              <Typography variant="body1">$300</Typography>
+            </div>
+          </CardContent>
+        </Card>
+        <div className={classes.Btn}>
+          <Link href="/0/rooms"> 
+          <Button
+            size="small"
+            className={classes.bookBtn}
+            variant="contained"
+            className={classes.editBtn}
+          >
+            Edit
+          </Button>
+          </Link>
+          <Link href="/">
```

Comment:
I think instead of redirect user to the home page after submitting a request, you should popup a toaster to notify the status of request if it's succeed or failed.

Reply:
thanks resolved